src/core/config.go

```diff
@@ -209,6 +209,7 @@ func DefaultConfiguration() *Configuration {
 	config.Please.Autoclean = true
 	config.Please.DownloadLocation = "https://get.please.build"
 	config.Please.NumOldVersions = 10
+	config.Please.NumThreads = runtime.NumCPU() + 2
```

Comment:
Does this work appropriately in Docker containers? Would GOMAXPROCS be more appropriate?

Reply:
Docs say runtime.NumCPU is "the number of logical CPUs usable by the current process" which implies it might be aware of cgroups etc; I think it is probably the best option (GOMAXPROCS is probably going to end up using it anyway)